src/ucp/core/ucp_worker.c

```diff
@@ -534,12 +534,19 @@ ucs_status_t ucp_worker_wait(ucp_worker_h worker)
         return status;
     }
 
+    status = ucp_worker_arm(worker);
```

Comment:
please also update UCP documentation:
- for 'ucp_worker_arm' - that it is not needed before 'wait' , just for get_efd
- that UCP_FEATURE_WAKEUP is needed to use this wait/get_efd/arm functions